src/controller/scan/base_controller.go

```diff
@@ -86,6 +87,8 @@ type basicController struct {
 
 	execMgr task.ExecutionManager
 	taskMgr task.Manager
+	// Converter for V1 report to V2 report
+	reportV1ToV2Converter postprocessors.NativeScanReportConverter
```

Comment:
Could rename `reportV1ToV2Converter` to `reportConverter` ? There is no report which is v2.

We only have report v1.0 and v1.1.

Reply:
Dpne